src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -93,7 +93,12 @@ public void setReplay(Replay replay) {
     timeLabel.setText(timeService.asShortTime(replay.getStartTime()));
     modLabel.setText(replay.getFeaturedMod().getDisplayName());
     playerCountLabel.setText(i18n.number(replay.getTeams().values().stream().mapToInt(List::size).sum()));
-    qualityLabel.setText(i18n.get("percentage", (int) ratingService.calculateQuality(replay) * 100));
+    double gameQuality = ratingService.calculateQuality(replay);
+    if (Double.isNaN(gameQuality)) {
```

Comment:
Should it not be if(!Double.isNaN(gameQuality))

Reply:
did you test this at all?

Reply:
Yeah, I was way to sleepy when I did that

Reply:
I tested it after fixing the cast before adding the NaN check, lesson lear ned, don't change a PR minuted before going to bed.